bash_conditionals_loops/bash_conditionals_loops.md

```diff
@@ -531,10 +571,17 @@ do
 done
 ```
 
-This loop also looks at every `.txt` file in the current directory, but the `if` statement is doing a few new things. You might have figured out from running the code that `[[ "$file" == *bear* ]]` is checking whether the file's **name** contains the string `bear`, possibly with other characters before or after. The double equals sign `==` is the test function for whether or not two strings are equal. But what are the double brackets doing?
+This loop also looks at every `.txt` file in the current directory, but the `if` statement is doing a few new things. You might have figured out from running the code that `[[ "$file" = *bear* ]]` is checking whether the file's **name** contains the string `bear`, possibly with other characters before or after. But what are the double brackets doing?
 
 Double brackets are an extension of single square brackets, with more powerful tools, like the pattern matching we are doing in this example with the character wildcard `*`. Since they extend the uses of single brackets, sometimes double and single brackets do the same thing, and sometimes using one type or the other is required. There are strongly held opinions on [which is better](https://stackoverflow.com/questions/669452/are-double-square-brackets-preferable-over-single-square-brackets-in-b) when you have a choice, but day-to-day your best course of action is to copy bracket style when you look up how to code a particular action.
 
+<div class = "options">
+<b style="color: rgb(var(--color-highlight));">Another option</b><br>
```

Comment:
```suggestion
<div class = "help">
<b style="color: rgb(var(--color-highlight));">Troubleshooting help</b><br>
```

I think this box is great, but it feels more like a help box than options to me. Feel free to push back, of course.

Reply:
good call!